openml-provider-archetype/src/main/resources/archetype-resources/src/main/java/openml/MyOpenmlModelLoader.java

```diff
@@ -0,0 +1,57 @@
+package ${groupId}.openml;
+
+import com.feedzai.openml.data.schema.DatasetSchema;
+import com.feedzai.openml.provider.descriptor.MLAlgorithmDescriptor;
+import com.feedzai.openml.provider.descriptor.fieldtype.ParamValidationError;
+import com.feedzai.openml.provider.exception.ModelLoadingException;
+import com.feedzai.openml.provider.model.MachineLearningModelLoader;
+import com.feedzai.openml.util.load.LoadSchemaUtils;
+import com.feedzai.openml.util.validate.ValidationUtils;
+import com.google.common.collect.ImmutableList;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+public class MyOpenmlModelLoader implements MachineLearningModelLoader<MyOpenmlModel> {
+
+    private MLAlgorithmDescriptor mlAlgorithmDescriptor;
+
+    public MyOpenmlModelLoader(final MLAlgorithmDescriptor mlAlgorithmDescriptor) {
+        this.mlAlgorithmDescriptor = mlAlgorithmDescriptor;
+    }
+
+    @Override
+    public MyOpenmlModel loadModel(final Path modelPath, final DatasetSchema schema) throws ModelLoadingException {
```

Comment:
why not add the validation methods that are being used by every model loader?

basically all the model loader are following this structure:



        logger.debug("Loading model from {}.", modelPath);
        ClassificationValidationUtils.validateParamsModelToLoad(this, modelPath, schema, ImmutableMap.of());
        ...

        ClassificationValidationUtils.validateClassificationModel(schema, modelLoaded);

        logger.info("Model successfully loaded [{}]", modelPath);
        return modelLoaded;

Reply:
As stated on the PR description, the goal of these classes is not to be 100% correct and bulletproof, but more like a starting point to new providers. As such, I do prefer to keep this simple instead of more correct. Maybe we could add some small comments saying that 'proper' models should check specific details that we are already aware of